src/Map/src/MapElement.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map;
+
+interface MapElement
```

Comment:
I believe it's enough, especially when you have `Elements` class that contains a list of `(Map)Element`:
```suggestion
interface Element
```